x-pack/plugins/stack_connectors/server/connector_types/servicenow_itom/api.test.ts

```diff
@@ -15,7 +15,7 @@ describe('api_itom', () => {
   let externalService: jest.Mocked<ExternalServiceITOM>;
   const eventParamsWithFormattedDate = {
     ...itomEventParams,
-    time_of_event: '2021-10-13, 10:51:44',
+    time_of_event: '10/13/2021, 10:51:44',
```

Comment:
Node.js v18.13.0 ships with a newer version of ICU (i18n library), which changed how `en-CA` dates are rendered. I'm not sure if this is an issue for upgrading? For details see: https://github.com/nodejs/node/issues/45945

Reply:
I think we should change the code here `x-pack/plugins/stack_connectors/server/connector_types/servicenow_itom/api.ts` to keep the desired format `yyyy-MM-dd HH:mm:ss GMT` and leave the test as it was before. Could you please change from `en-CA` to `sv-SE`. I will open an issue to find a way to format the day without relying on the `toLocaleDateString` function.

Reply:
Here's a link to the commit with my change: https://github.com/elastic/kibana/pull/144012/commits/5e98895878cc7e011209acf7c9b33c9850944c64

Reply:
Thanks, @watson! According to the [ITOM docs](https://docs.servicenow.com/bundle/rome-it-operations-management/page/product/event-management/task/send-events-via-web-service.html) what you did is more inline with the expected format for the `time_of_event` attribute.

Reply:
Ha, even better then! Thanks :)